ml/src/svm.cpp

```diff
@@ -42,12 +42,16 @@
 #include <pcl/common/utils.h> // pcl::utils::ignore
 #include <pcl/ml/svm.h>
 
+#include <algorithm>
+#include <cctype>
+#include <cfloat>
 #include <climits>
 #include <cmath>
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+#include <iostream>
```

Comment:
Please check if the added headers are actually required: cctype, cfloat, iostream